chord_metadata_service/mohpackets/data/data_converter.py

```diff
@@ -141,29 +142,40 @@ def replace_values(input_data, transformation_rules):
     return input_data
 
 
-def main():
-    print("Select an option:")
-    print("1. Convert small dataset")
-    print("2. Convert medium dataset")
-    print("3. Convert large dataset")
-    print("4. Exit")
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--size',
+        type=str,
+        default='s',
+        choices=['s', 'm', 'l'],
+        help="Size of the dataset to convert (default: small)"
+    )
+    parser.add_argument(
+        '--program',
+        type=str,
+        default='SYNTHETIC',
+        help="Program ID prefix (default: SYNTHETIC)"
+    )
+    args = parser.parse_args()
+    return args
 
-    choice = int(input("Enter your choice [1-4]: "))
 
-    if choice == 1:
+def main():
+    args = parse_args()
+
+    if args.size:
+        size_mapping = {'s': 'small', 'm': 'medium', 'l': 'large'}
+        path = f"{size_mapping[args.size]}_dataset"
+    else:
         path = "small_dataset"
-    elif choice == 2:
-        path = "medium_dataset"
-    elif choice == 3:
-        path = "large_dataset"
-    elif choice == 4:
-        print("Exiting...")
-        exit()
+
+    if args.program:
+        program_id = args.program
     else:
-        print("Invalid option. Please try again.")
-        return
+        program_id = "SYNTHETIC"
```

Comment:
I think that you don't need the 'if/else' statements here because the default values mean these args will always be present.

Reply:
Fixed.